source/functions/BEMIO/waveNumber.m

```diff
@@ -63,31 +63,31 @@
     v = 0;  %current velocity in y direction [m/s]
 % end
 
```

Comment:
Can we remove `waveNumber` from `waveClass` and only call on this? How do they differ?

Reply:
This BEMIO functions contains a more complex iteration method that also considers wave current effects on the wavenumber. However this BEMIO function is only called by Read_NEMOH() for the mean drift calculations, and it never actually inputs current data. I replaced this with the waveClass function because it is more clear and should converge to the same value.